js/editor-ta.js

```diff
@@ -89,8 +110,8 @@ EditorTextArea.prototype.calibrateDimensions = function() {
 }
 
 /**
- * Updates height for both line number gutter and textarea.
  * @param {number} height
+ * Updates height for both line number gutter and textarea.
```

Comment:
if you're updating these to be consistent throughout the file I think it's better to do it the other way and be consistent with google style guide

Reply:
will switch it :)